CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 # GitHub Theme Changelog
 
 ## [Unreleased]
+Fix Java colors for 2021
```

Comment:
also update https://github.com/jiyuan0125/github-theme-for-jetbrains/blob/16bf1de6b2c5806a953516ce83121f5116132d14/gradle.properties#L6 this to 1.1.4

Reply:
also please update exact changes under unreleased section as fix should not be for 2021 only